apps/src/turtle/blocks.js

```diff
@@ -259,6 +259,28 @@ exports.install = function (blockly, blockInstallOptions) {
     }
   };
 
+blockly.Blocks.point_to = {
+    // Block for pointing to a specified direction
+    helpUrl: '',
+    init: function () {
+      this.setHSV(184, 1.00, 0.74);
+      this.appendDummyInput()
+      .appendTitle(msg.pointTo());
```

Comment:
nit: indent this line like you did below

Reply:
Fixed!  Thanks